htdocs/install/mysql/tables/llx_bookcal_availabilities_extrafields.key.sql

```diff
@@ -0,0 +1,19 @@
+-- Copyright (C) 2023 Alice Adminson <aadminson@example.com>
```

Comment:
The name of the table must be 
llx_bookcal_availabilities_extrafields-bookcal.key.sql